force-app/main/default/lwc/givingSummary/givingSummary.js

```diff
@@ -0,0 +1,73 @@
+import { api, LightningElement } from 'lwc';
+import donorsGivenSummary from '@salesforce/label/c.donorsGivenSummary';
+import donorsLifetime from '@salesforce/label/c.donorsLifetime';
+import donorsThisYear from '@salesforce/label/c.donorsThisYear';
+import donorsPreviousYear from '@salesforce/label/c.donorsPreviousYear';
+import FORM_FACTOR from '@salesforce/client/formFactor';
+import getDonationsSummary from '@salesforce/apex/GivingSummaryController.getDonationsSummaryForContact'
+const FormFactorType = Object.freeze({
```

Comment:
`FormFactorType` should be moved into the GivingSummary class instead of being a global variable?

Reply:
That's just an enum, I dont think it has to be a class members(the enum)